PYME/LMVis/view_clipping_pane.py

```diff
@@ -231,14 +244,22 @@ def OnMouseScrollEvent(self, evt):
 
     def OnLeftDown(self, event):
         x = event.GetX()
-        #y = event.GetY()
+        y = event.GetY()
 
         x0p, wp, x0vp, xmvp, w, xo = self._get_coords()
+        maxy = self.Size[1] - self.ll_ctrl.Size[1]/2 + 2
     
         #hit test the limits
+        # check for clipping update via text control
```

Comment:
It's not very clear where you are checking for hits here ...

Reply:
A comment describing where you are checking would be useful

Reply:
Also, to be consistent with buttons and other clickable controls, we should be showing the text control on the mouse button release (EVT_LEFT_UP) rather than the press, but only if we were not dragging the bounds.

Reply:
Sure thing. On the dragging front: I originally wanted to put the test for displaying the textbox after the dragging, but I'll have to make sure `y < self.Size[1] - self.ll_ctrl.Size[1]` on each of the dragging tests. Which I suppose is  about as acceptable as what I have here.